src/chain/chain.rs

```diff
@@ -371,22 +371,20 @@ impl Chain {
         let our_section_size = self.state.our_joined_members().count();
         let safe_section_size = self.safe_section_size();
 
+        // As a measure against sybil attacks, we don't increment the age counters on infant churn
+        // once we reach safe_section_size.
         if our_section_size >= safe_section_size
```

Comment:
just to confirm that:
1, when `section_size < safe_section_size`, infant_churn causes age increment
2, non-infant churn always causes aging
right ?

Reply:
That's right.